### PR TITLE
integrations: Add notes for Facebook.

### PR DESCRIPTION
**Pull request by MadElf1337**

### CODE REVIEW

File: webhook-integrations/Facebook/notes.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Facebook Integration Investigation
+
+[Facebook](https://www.developers.facebook.com) is a social networking service that also allows developers
+to track analytics of their apps.
```
Comment (roberthoenig) [inline webhook-integrations/Facebook/notes.md]: Is it only for apps, or can we also receive user analytics?

File: webhook-integrations/Facebook/notes.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+e.g.
+If I configure my App to use the Facebook SDK, & configure the parameter 'User' for sending out 
+notifications & I subscribe to the 'email' case in 'User', then Facebook sends out a notification
+to the webhook every time a new user email is added/deleted from the App.
+
+## Links
+Facebook For Developers: https://developers.facebook.com
+
+## Other Notes
+The 4 parameters were chosen considering their use if integrated with Zulip, & the cases for each
```
Comment (roberthoenig) [inline webhook-integrations/Facebook/notes.md]: What exactly are these 4 parameters? If you mean "4 different webhook notification types", you should specify them individually.

File: webhook-integrations/Facebook/notes.md (hunk 1 above)
Comment (MadElf1337) [reply webhook-integrations/Facebook/notes.md]: What do you mean by we?
It is for apps & websites

Comment (MadElf1337) [reply webhook-integrations/Facebook/notes.md]: Also, it is meant only for the developers, no user can view these analytics

Comment (roberthoenig) [reply webhook-integrations/Facebook/notes.md]: Okay. I was asking if it is also possible to receive analytical data for some Facebook user, e.g. receive a notification whenever user x posts a status update.

Comment (MadElf1337) [reply webhook-integrations/Facebook/notes.md]: no, should I mention that in the notes?

Comment (eeshangarg) [reply webhook-integrations/Facebook/notes.md]: @MadElf1337: Probably mention that in the notes yes. :)

**Comment by eeshangarg** [comment]
@MadElf1337: Merged! Great work! :)

**Pull request merged**